apps/meeting/server.js

```diff
@@ -60,24 +76,34 @@ const server = require(protocol).createServer(
         const name = query.name;
         const region = query.region || 'us-east-1';
 
+        let client = getClientForMeeting(meetingCache[title]);
```

Comment:
This is not useful if query param has `query.ns_es === 'true'`, so can this be moved to else block?

Reply:
i keep it the same as https://github.com/aws/amazon-chime-sdk-js/blob/master/demos/browser/server.js#L74-L91. client is used to createMeeting and createAttendee. which else block are you referring to?